### PR TITLE
[TEVA-1757] Job application confirmation page

### PR DESCRIPTION
**Pull request by david-mears-dfe**
## Jira ticket URL

https://dfedigital.atlassian.net/browse/TEVA-1757

## Changes in this PR:

- Reinstate panel styles
- Changes to notification component to allow us to customize the html attributes when using notification styles in a non-notification context, and to fix its styles
- Create application confirmation page, with application feedback

## Screenshots 

Screenshot 2021-02-18 at 19 21 21

Screenshot 2021-02-18 at 19 21 25

Screenshot 2021-02-18 at 19 21 42

Screenshot 2021-02-18 at 19 21 55

### CODE REVIEW

**Comment by martin-bangoura** [comment]
Review app deployed to <https://teaching-vacancies-review-pr-2909.london.cloudapps.digital>

File: app/components/shared/notification_component.rb [hunk 1]
```diff
@@ -1,17 +1,24 @@
 class Shared::NotificationComponent < ViewComponent::Base
-  def initialize(content:, style:, links: nil, dismiss: true, background: false, alert: "warning")
+  def initialize(content:, style:,
```
Comment (cpjmcquillan) [inline app/components/shared/notification_component.rb:2]: I think we could benefit from a wholesale refactor of our approach to shared components like this (e.g passing custom classes and html_attributes consistently) now that we have more experience using them and their documentation has improved significantly. It's outside the scope of this ticket though so I'm happy for us to do this temporarily to achieve what you're trying to - and I'll make a ticket about what I want us to do eventually

File: config/locales/jobseekers.yml [hunk 2]
```diff
@@ -143,8 +143,25 @@ en:
       review:
         title: Review your application
       submit:
-        panel_body: "We have sent an email confirmation to %{email}"
-        title: Your application has been submitted
+        feedback:
```
Comment (cpjmcquillan) [inline config/locales/jobseekers.yml:146]: Can we alphabetise stuff please?

**Review by cpjmcquillan — APPROVED** [review]
🚀  (pending updates to `jobseekers_can_submit_a_job_application_spec.rb`)

**Review by alexbowen — APPROVED** [review]

**Pull request merged**